include/mc_tasks/lipm_stabilizer/StabilizerTask.h

```diff
@@ -742,6 +842,30 @@ struct MC_TASKS_DLLAPI StabilizerTask : public MetaTask
   /**< Whether the estimator needs to be reset (robot in the air, initialization) */
   bool dcmEstimatorNeedsReset_ = true;
 
+  /** @name Members related to stabilization in the presence of external wrenches
+   *
+   *  Adding an offset to the CoM for the predictable / measurable external wrenches on the robot surface.
+   *  @{
+   */
+  std::vector<std::pair<std::string, sva::ForceVecd>> extWrenchesTarget_; /**< Target (expected) external wrenches */
+  std::vector<std::pair<std::string, sva::ForceVecd>> extWrenchesMeasured_; /**< Measured external wrenches */
```

Comment:
Why not merge the two into `std::vector<std::tuple<std::string, sva::ForceVecd, sva::ForceVecd>>>` or introduce an intermediary struct to hold the external wrench data?

Reply:
Updated in https://github.com/jrl-umi3218/mc_rtc/pull/95/commits/673616e83e8fbf9f2350e8b311d50ebcb5aafbbd
But I couldn't implement it very elegantly (for example, function in the argument: https://github.com/jrl-umi3218/mc_rtc/pull/95/commits/673616e83e8fbf9f2350e8b311d50ebcb5aafbbd#diff-59c59e7e81d4b42954dde739045b64f14846c73ff9dbf842e3b840609582f485R717)

Reply:
I've tried a different approach in 9e38fce04 that should have no overhead

Reply:
Thanks for a solution I could not have imagined!